networktable.c

```diff
@@ -39,23 +38,40 @@ bool create_network_table(void)
 	return true;
 }
 
-// Read kernel's ARP cache using procfs
-void parse_arp_cache(void)
+bool create_network_addresses_table(void)
+{
+	bool ret;
+	// Create network_addresses table in the database
+	ret = dbquery("CREATE TABLE network_addresses ( id INTEGER NOT NULL, " \
+	                                               "ip TEXT NOT NULL, "\
+	                                               "lastQuery INTEGER NOT NULL, " \
+	                                               "UNIQUE(id,ip));");
+	if(!ret){ dbclose(); return false; }
+
+	// Update database version to 5
+	ret = db_set_FTL_property(DB_VERSION, 5);
+	if(!ret){ dbclose(); return false; }
+
+	return true;
+}
+
+// Parse kernel's neighbor cache
+void parse_neigh_cache(void)
```

Comment:
For three extra letters, we can use the full word "neighbor" in the function name, and it may be less confusing.